src/plugins/profiling/scripts/export_from_testing.sh

```diff
@@ -35,10 +35,11 @@ export_events() {
   local from=$2
   local to=$3
 
-  docker run --rm -ti --net=host -v "$PWD:/data" -w /data elasticdump/elasticsearch-dump:latest \
+  docker run --rm -ti --net=host -v "$PWD:/data" -w /data -e "NODE_OPTIONS=--max_old_space_size=8192" \
+    elasticdump/elasticsearch-dump:latest \
     --input="https://""${ELASTIC_TESTING_CREDENTIALS}""@profiling-es.35.240.6.93.ip.es.io/${index}" \
     --output="${index}-data_${from}_${to}.json.gz" \
-    --type=data  --fsCompress --noRefresh --limit=10000 --support-big-int \
+    --type=data  --fsCompress --noRefresh --limit=100000 --support-big-int \
```

Comment:
the limit settings will require a separate PR (for which I have a branch locally) to include that in the index mappings applied to the env

Reply:
Awesome, any concerns that keep you from creating a PR ?